api/groups.go

```diff
@@ -66,3 +66,51 @@ func (api *API) AddUserToGroupHandler(ctx context.Context, w http.ResponseWriter
 
 	return models.NewSuccessResponse(jsonResponse, http.StatusOK, nil), nil
 }
+
+//RemoveUserFromGroupHandler adds a user to the specified group
+func (api *API) RemoveUserFromGroupHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	vars := mux.Vars(req)
+	group := models.Group{Name: vars["id"]}
```

Comment:
Are we always certain that the`.../{id}/...`  param passed to the handler represents a valid group?